content/webapp/components/ContentSearchResult/ContentSearchResult.tsx

```diff
@@ -0,0 +1,100 @@
+import NextLink from 'next/link';
+import { FunctionComponent } from 'react';
+import styled from 'styled-components';
+
+import linkResolver from '@weco/common/services/prismic/link-resolver';
+import { font } from '@weco/common/utils/classnames';
+import { HTMLDate } from '@weco/common/views/components/HTMLDateAndTime';
+import Space from '@weco/common/views/components/styled/Space';
+import DateRange from '@weco/content/components/DateRange/DateRange';
+
+type Props = {
+  uid: string | null;
+  type: string;
+  title: string;
+  description?: string;
+  highlightTourType?: 'audio' | 'bsl';
+  tags?: string[];
+  dates?: { start: string; end?: string };
+  times?: { start: string; end: string };
+  contributors?: string;
+};
+
+const Link = styled(NextLink)`
+  text-decoration: none;
+  color: inherit;
+  display: block;
+`;
+
+const Type = styled(Space).attrs({
+  $v: { size: 's', properties: ['margin-bottom'] },
+  className: font('intr', 6),
+})``;
```

Comment:
np; Not styled and not used more than once so no SC needed?

Reply:
makes the markup look neater though?

Reply:
I agree when it clarifies what the element does, it's nice. I'm just referring to what we'd landed on for SC for code review, but it's not a blocker ofc, very much a np! Maybe we'll see if we'd like to flex those rules a little in a couple months?